src/queries/query-keys.ts

```diff
@@ -1,3 +1,4 @@
 export const QUERY_KEYS = {
   GET_CATEGORIES: () => ['GET_CATEGORIES'],
+  GET_PRODUCT: (productId: number) => ['GET_PRODUCT', productId],
```

Comment:
전체 상품 리스트를 불러오는 api가 `/product`이므로 '상품 개별 조회' api라는 의미로 변경해야 할 것 같습니다.
```suggestion
  GET_PRODUCTS_DETAIL: (productId: number) => ['GET_PRODUCTS_DETAIL', productId],
```